claude/constants.py

```diff
@@ -83,9 +83,11 @@ def __str__(self) -> str:
 
 ###### Models
 class Model(str, Enum):
-    CLAUDE_2 = "claude-2"
-    CLAUDE_1_P_3 = "claude-1.3"
-    CLAUDE_INSTANT_100k = "claude-instant-100k"
+    CLAUDE_2_P_0 = "claude-2.0"
+    CLAUDE_2_P_1 = "claude-2.1"
+    CLAUSE_INSTANT_1_P_2 = "claude-instant-1.2"
```

Comment:
Nit: Write out the whole word "point"

Reply:
Convention so far is `X_P_Y`. I'm fine with this.

Reply:
Should we write a follow up PR to fix this? This is less accessible and worse for i18n code readability. My quant told me he was confused (he's from China)

Reply:
Feel free to write a followup for this one lil bro.